### ISSUE TITLE
Replays.tv isn't recording my games 

### ISSUE THREAD

**Issue by leonardoCrosa** [lulzsun/RePlaysTV#31]
Hello,

So I play league of legends, I used to have Plays.tv, and recently i found out Replays exists.

I already had Plays installed and I installed the 3.0.3a Version on top of it. My old replays appeared after the installation. Buy my new games didn't record or show up. (Automatic recording is enabled)

Then i tried uninstalling Plays and deleting everything from the appdata Plays and Plays-ltc folders. Then downloaded and installed Plays from the wayback machine link and re installed Replays. And I followed the video explanation. (When installing Replays i wait untill a sign appears saying its done and i can close the installer) The program works fine but it still isnt recording.

Also when I press the Clip or Record keys the game freezes for a milisec as if it started recording or saved a clip. But still nothing is saved. 

I also tried changing the save folder, but to no success.

I would appreciate any help given to fix this particular problem i have, since i don't like any other recording apps.

Thanks in advance,

Leonardo

**Comment by JayBoom** [comment]
Yeah lots of comments on discord and reddit saying League of Legends isn't recording, I really hope all the games not recording will be fixed soon.
